easyaudit/signals/model_signals.py

```diff
@@ -68,7 +68,7 @@ def pre_save(sender, instance, raw, using, update_fields, **kwargs):
 
             # created or updated?
             if not created:
-                old_model = sender.objects.get(pk=instance.pk)
+                old_model = sender._default_manager.get(pk=instance.pk)
```

Comment:
_base_manager i think

Reply:
https://docs.djangoproject.com/en/4.0/topics/db/managers/#default-managers-1, https://docs.djangoproject.com/en/4.0/topics/db/managers/#base-managers

can someone verify the most correct solution?

Reply:
It should be _base_manager which will always return all records, the _default_manager may be filtering out records and can still raise an DoesNotExist exception.